sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcFilters.scala

```diff
@@ -67,6 +67,16 @@ private[sql] object OrcFilters {
     }
   }
 
+  // Since ORC 1.5.0 (ORC-323), we need to quote for column names with `.` characters
+  // in order to distinguish predicate pushdown for nested columns.
+  private def quoteAttributeNameIfNeeded(name: String) : String = {
+    if (!name.contains("`") && name.contains(".")) {
```

Comment:
Does this condition take the backtick in column name into account? For instance,

```
>>> spark.range(1).toDF("abc`.abc").show()
+--------+
|abc`.abc|
+--------+
|       0|
+--------+
```

Reply:
Thank you for review. I'll consider that, too.

Reply:
@HyukjinKwon . Actually, Spark 2.3.2 ORC (native/hive) doesn't support a backtick character in column names.  It fails on **writing** operation. And, although Spark 2.4.0 broadens the supported special characters like `.` and `"` in column names, the backtick character is not handled yet.

So, for that one, I'll proceed in another PR since it's an improvement instead of a regression.

Also, cc @gatorsmile and @dbtsai .

Reply:
For `ORC` and `AVRO` improvement, [SPARK-25722](https://issues.apache.org/jira/browse/SPARK-25722) is created.